sw/device/silicon_creator/lib/sigverify/rsa_verify.c

```diff
@@ -203,14 +202,12 @@ rom_error_t sigverify_rsa_verify(const sigverify_rsa_buffer_t *signature,
   sigverify_rsa_buffer_t enc_msg;
   rom_error_t error = kErrorSigverifyBadRsaSignature;
   switch (use_sw) {
-    case kHardenedBoolTrue:
-      error = sigverify_mod_exp_ibex(key, signature, &enc_msg);
-      break;
     case kHardenedBoolFalse:
       error = sigverify_mod_exp_otbn(key, signature, &enc_msg);
       break;
     default:
-      HARDENED_TRAP();
```

Comment:
This change goes away if we refactor `sigverify_use_sw_rsa_verify()`

Reply:
see comment above.